proto/lavanet/lava/dualstaking/query.proto

```diff
@@ -14,6 +15,11 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/lavanet/lava/dualstaking/params";
   }
+
+  // Queries a list of ListProjects items.
+	rpc DelegatorProviders(QueryDelegatorProvidersRequest) returns (QueryDelegatorProvidersResponse) {
```

Comment:
indentation in proto should be 2 spaces

Reply:
done. see https://github.com/lavanet/lava/commit/f64a33890597cbbf9002d2b66c3daf435abfe569